pennylane/gradients/jvp.py

```diff
@@ -274,7 +309,56 @@ def batch_jvp(tapes, tangents, gradient_fn, shots=None, reduction="append", grad
         to tapes with no trainable parameters.
 
     **Example**
-    # TODO: add examples
+
+    .. code-block:: python
+
+        import jax
+        qml.enable_return()
+        x = jax.numpy.array([[0.1, 0.2, 0.3],
+                             [0.4, 0.5, 0.6]])
+
+        def ansatz(x):
+            qml.RX(x[0, 0], wires=0)
+            qml.RY(x[0, 1], wires=1)
+            qml.RZ(x[0, 2], wires=0)
+            qml.CNOT(wires=[0, 1])
+            qml.RX(x[1, 0], wires=1)
+            qml.RY(x[1, 1], wires=0)
+            qml.RZ(x[1, 2], wires=1)
+
+        with qml.tape.QuantumTape() as tape1:
+            ansatz(x)
+            qml.expval(qml.PauliZ(0))
+            qml.probs(wires=1)
+
+        with qml.tape.QuantumTape() as tape2:
+            ansatz(x)
+            qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+
+        tapes = [tape1, tape2]
+
+    Both tapes share the same circuit ansatz, but have different measurement outputs.
+
+    We can use the ``batch_jvp`` function to compute the Jacobian vector product,
+    given a list of tangents ``tangent``:
+
+    >>> tangent_0 = [jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0)]
+    >>> tangent_1 = [jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0)]
+    >>> tangents = [tangent_0, tangent_1]
+    >>> vjp_tapes, fn = qml.gradients.batch_vjp(tapes, dys, qml.gradients.param_shift)
```

Comment:
What is `dys` here?

Reply:
I think I removed this line on the latest commit :)